QLNhapCu/src/main/java/org/example/LaborExport.java

```diff
@@ -0,0 +1,29 @@
+package org.example;
+
+import java.util.Date;
+
+class LaborExport extends Immigrant {
+    private String company;
+    private String laborType;
+
+    public LaborExport(String immigrantID, String name, Date birthDate, String passport, String nationality, Date immigrantDate, int stayLength, double discountRate, String company, String laborType) {
+        super(immigrantID, name, birthDate, passport, nationality, immigrantDate, stayLength, discountRate);
+        this.company = company;
+        this.laborType = laborType;
+    }
+
+    public String getCompany() {
+        return company;
+    }
+
+    public String getLaborType() {
+        return laborType;
+    }
+
+    @Override
+    public void showDetails() {
+        System.out.println("Labor Export Details:");
```

Comment:
thiếu in ra của những thuộc tính của immiogrant sử duyngj thêm method tostring immiogrant